### ISSUE TITLE
species classification using rgb data and weak labels

### ISSUE THREAD

**Issue by MarconiS** [weecology/DeepForest#97]
Using automatically detected species labels, pretrain deepForest on RGB data and fine-tune on field annotated crown ids. First application using a mid hard case study (HARV NEON site)
